services/galley/src/Galley/Cassandra/Client.hs

```diff
@@ -42,7 +42,7 @@ import UnliftIO qualified
 updateClient :: Bool -> UserId -> ClientId -> Client ()
 updateClient add usr cls = do
   let q = if add then Cql.addMemberClient else Cql.rmMemberClient
-  retry x5 $ write (q cls) (params LocalQuorum (Identity usr))
+  retry x5 . void $ trans (q cls) (params LocalQuorum (Identity usr))
```

Comment:
Not sure here for `addMemberClient`. Is it possible that a user has no clients, and this really is an upsert?

Reply:
not sure, you may be right 52d4470f6

Reply:
sorry, i missed a spot: ea8595e3a